koji_containerbuild/plugins/hub_containerbuild.py

```diff
@@ -35,6 +36,12 @@
 
 logger = logging.getLogger('koji.plugins')
 
+# update task signatures, so they can properly parse options for policies
+koji.tasks.LEGACY_SIGNATURES['buildContainer'] = [[['src', 'target', 'opts'],
+                                                  None, None, (None,)]]
+koji.tasks.LEGACY_SIGNATURES['buildSourceContainer'] = [[['target', 'opts'],
+                                                        None, None, (None,)]]
```

Comment:
shouldn't this be just : None, (None,)   ?

Reply:
No, basically order of fields is args/var_args/var_kw/defaults, so two Nones refers to missing var_args/var_kwargs. (https://pagure.io/koji/blob/master/f/koji/util.py#_285)

Reply:
ic